schema/Metrics.v1.0.0.schema.json

```diff
@@ -0,0 +1,354 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Have you verified this file is valid under Draft-07 of JSON Schema? There are some unfamiliar elements here, and I wonder if this is using an OpenAPI dialect? See https://json-schema.org/blog/posts/validating-openapi-and-json-schema

Reply:
This schema file is not strictly following draft-07 as it is using terms like ```x-ms-enum``` that are Microsoft specific. 

I updated the schema file to strictly follow draft-07 schema.